packages/preview-middleware-client/test/unit/cpe/init.test.ts

```diff
@@ -64,28 +79,31 @@ describe('main', () => {
     jest.spyOn(ui5Utils, 'getIcons').mockImplementation(() => {
         return mockIconResult;
     });
-    const attachSelectionChange = jest.fn().mockImplementation((newHandler: (event: Event) => Promise<void>) => {
-        return newHandler;
-    });
+    // const attachSelectionChange = jest.fn().mockImplementation((newHandler: (event: Event) => Promise<void>) => {
+    //     return newHandler;
+    // });
 
-    const rta = {
-        attachSelectionChange,
-        getSelection: jest.fn().mockReturnValue([{ setSelected: jest.fn() }, { setSelected: jest.fn() }]),
-        attachUndoRedoStackModified: jest.fn(),
-        getFlexSettings: jest.fn().mockReturnValue({ layer: 'VENDOR', scenario: common.SCENARIO.UiAdaptation }),
-        getRootControlInstance: jest.fn().mockReturnValue({
-            getManifest: jest.fn().mockReturnValue({ 'sap.app': { id: 'testId' } })
-        }),
-        attachStop: jest.fn(),
-        attachModeChanged: jest.fn()
-    } as any;
+    // const rta = {
+    //     attachSelectionChange,
+    //     getSelection: jest.fn().mockReturnValue([{ setSelected: jest.fn() }, { setSelected: jest.fn() }]),
+    //     attachUndoRedoStackModified: jest.fn(),
+    // getFlexSettings: jest.fn().mockReturnValue({ layer: 'VENDOR', scenario: common.SCENARIO.UiAdaptation }),
+    //     getRootControlInstance: jest.fn().mockReturnValue({
+    //         getManifest: jest.fn().mockReturnValue({ 'sap.app': { id: 'testId' } })
+    //     }),
+    //     attachStop: jest.fn(),
+    //     getService: jest.fn(),
+    //     attachModeChanged: jest.fn()
+    // } as any;
```

Comment:
Will the comments be remove or this is needed in the future?

Reply:
Removed 968665e